core/player/src/view/resolver/index.ts

```diff
@@ -185,21 +190,17 @@ export class Resolver {
 
     if (id) {
       if (this.idCache.has(id)) {
-        // Only log this conflict once to cut down on noise
-        // May want to swap this to logging when we first see the id -- which may not be the first render
         if (isFirstUpdate) {
           if (node.type === NodeType.Asset || node.type === NodeType.View) {
             this.logger?.error(
               `Cache conflict: Found Asset/View nodes that have conflicting ids: ${id}, may cause cache issues.`
             );
           } else if (node.type === NodeType.Value) {
             this.logger?.info(
-              `Cache conflict: Found Value nodes that have conflicting ids: ${id}, may cause cache issues. To improve performance make value node IDs globally unique.`
```

Comment:
i think the cache conflict here is important, it's the duplicate id that would be causing potential issues

Reply:
So i have removed the cache conflict as this is providing an info and not the error message. In the error we are already giving the same info ... Just a thought that how would both the error and the info be the same. Nevertheless, I can revert it.